code/game/objects/items/storage/belt.dm

```diff
@@ -781,6 +781,8 @@
 	item_state = "ncr_bandolier"
 
 //Regular Quiver
+//MOVING TO F13STORAGE
```

Comment:
If you moved this into Coyote_modular you wouldnt have had to comment out the code <3
It would have been overwritten entirely.